src/main/java/org/galatea/starter/entrypoint/SettlementRestController.java

```
@@ -85,7 +85,7 @@
       return ResponseEntity.ok(msn.get());
     }
 
-    return new ResponseEntity<>(HttpStatus.NOT_FOUND);
+    throw new SettlementMissionNotFoundException(id);
```

I don't agree that if we don't find an object in the database we should throw an exception. Exceptions should represent issues in the execution of the application that were unexpected and that workflow has to stop.